setup.py

```diff
@@ -64,8 +64,8 @@ def run_tests(self):
     raise Exception('this library does not support python >2 and <3.5')
```

Comment:
it was using pythonesque boolean logic, but your suggestion of "This library does not support Python 3 versions below 3.5." is much, much better!